releasenotes/notes/add-more-sabre-trials-9b421f05d2f48d18.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features_transpiler:
+  - |
+    The :class:`.SabreLayout` transpiler pass has been updated to run an
+    additional 2 or 3 layout trials by default independently of the
+    ``layout_trials`` keyword argument's value. For all backends a trivial
+    layout and it's reverse are included just like the :class:`.DenseLayout`
+    trial that were added in 1.2.0. In addition if the backends are 127,
+    133, or 156 qubits the largest rings on an IBM backend heavy hex
+    connectivity graph are added. On those backends which are commonly run
+    this can provide a good starting point for some circuits on these
+    backends and for all others it's just an additional random trial.
```

Comment:
```suggestion
    ``layout_trials`` keyword argument's value. A trivial
    layout and its reverse are included for all backends, just like the :class:`.DenseLayout`
    trial that was added in 1.2.0. In addition to this, the largest rings on 
    an IBM backend heavy hex connectivity graph are added if the backends are 127,
    133, or 156 qubits. This can provide a good starting point for some circuits on these commonly run
    backends, while for all others it's just an additional "random trial".
```